.github/workflows/lifecycle-image.yml

```diff
@@ -17,6 +17,33 @@ jobs:
       - name: Publish to Registry
         run: |
           LIFECYCLE_VERSION=`echo ${{ github.event.release.tag_name }} | cut -d "v" -f2`
-          curl -o -L lifecycle.tgz https://github.com/buildpacks/lifecycle/releases/download/v${LIFECYCLE_VERSION}/lifecycle-v${LIFECYCLE_VERSION}+linux.x86-64.tgz
-          echo ${{ secret.DOCKER_PASSWORD }} | docker login -u ${{ secrets.DOCKER_USERNAME }} --password-stdin
-          go run ./tools/image/main.go -lifecyclePath ./lifecycle.tgz -tag buildpacksio/lifecycle:${LIFECYCLE_VERSION} -tag buildpacksio/lifecycle:latest
+          curl -o lifecycle.tgz -L https://github.com/buildpacks/lifecycle/releases/download/v${LIFECYCLE_VERSION}/lifecycle-v${LIFECYCLE_VERSION}+linux.x86-64.tgz
+          echo ${{ secrets.DOCKER_PASSWORD }} | docker login -u ${{ secrets.DOCKER_USERNAME }} --password-stdin
+          go run ./tools/image/main.go -lifecyclePath ./lifecycle.tgz -tag buildpacksio/lifecycle:${LIFECYCLE_VERSION}-linux -tag buildpacksio/lifecycle:latest-linux
+  build-lifecycle-image-windows:
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up go
+        uses: actions/setup-go@v2-beta
+        with:
+          go-version: '1.14'
+      - name: Publish to Registry
+        run: |
+          $LIFECYCLE_VERSION=$(echo ${{ github.event.release.tag_name }} | cut -d "v" -f2)
+          curl -o lifecycle.tgz -L https://github.com/buildpacks/lifecycle/releases/download/v${LIFECYCLE_VERSION}/lifecycle-v${LIFECYCLE_VERSION}+windows.x86-64.tgz
+          echo ${{ secrets.DOCKER_PASSWORD }} | docker login -u ${{ secrets.DOCKER_USERNAME }} --password-stdin
+          go run ./tools/image/main.go -lifecyclePath ./lifecycle.tgz -tag buildpacksio/lifecycle:${LIFECYCLE_VERSION}-windows -tag buildpacksio/lifecycle:latest-windows
+  create-lifecycle-image-manifest-list:
+    needs: [build-lifecycle-image-linux, build-lifecycle-image-windows]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Create Manifest Life
+        run: |
+          LIFECYCLE_VERSION=`echo ${{ github.event.release.tag_name }} | cut -d "v" -f2`
+          echo ${{ secrets.DOCKER_PASSWORD }} | docker login -u ${{ secrets.DOCKER_USERNAME }} --password-stdin
+          DOCKER_CLI_EXPERIMENTAL=enabled docker manifest create buildpacksio/lifecycle:0.99.20 buildpacksio/lifecycle:0.99.20-linux buildpacksio/lifecycle:0.99.20-windows
+          DOCKER_CLI_EXPERIMENTAL=enabled docker manifest push buildpacksio/lifecycle:0.99.20
```

Comment:
```suggestion
          DOCKER_CLI_EXPERIMENTAL=enabled docker manifest create buildpacksio/lifecycle:${LIFECYCLE_VERSION} buildpacksio/lifecycle:${LIFECYCLE_VERSION}-linux buildpacksio/lifecycle:${LIFECYCLE_VERSION}-windows
          DOCKER_CLI_EXPERIMENTAL=enabled docker manifest push buildpacksio/lifecycle:${LIFECYCLE_VERSION}
```

Reply:
🤦  good catch...

Pulling in the commit suggestion seems to have broken DCO... do you know how to fix?

Reply:
You need to pull down the change `git rebase` with `--signoff` and force push it. I have definitely done this before, you can add suggestions w/o breaking DCO by copying you signoff into the commit in the UI.